app/views/releases/_release.html.erb

```diff
@@ -14,7 +14,11 @@
       <%= github_user_avatar(user) %>
     <% end %>
   </td>
-  <td width="100%"><%= deployed_or_running_list @stages, release.version %></td>
+  <td width="100%">
+    <% cache ["deployed-stages", release, @stages] do %>
```

Comment:
Do we want to add an `expires_in` option to this?

Reply:
There should be no need for that – if either `release` or any of `@stages` get updated, the cache key will change.

Reply:
Ah and `stage` would get updated if a deploy's `job` is updated. Sounds good 👍